storagemarket/lp2pimpl/net.go

```diff
@@ -288,5 +291,7 @@ func (p *DealProvider) getDealStatus(req types.DealStatusRequest) types.DealStat
 			PublishCid:        pds.PublishCID,
 			ChainDealID:       pds.ChainDealID,
 		},
+		Transfer:       pds.Transfer,
```

Comment:
The transfer may contain sensitive information, eg the Authorization header, so we may not want to include it here

Reply:
Replaced just with TransferSize, so that we display progress in the command output.